### PR TITLE
INT-536_remove_aws_config

### PR DESCRIPTION
**Pull request by fernau**
### Description
Hazelcast configuration is no longer updated in the default recipe when building docker images.
### Jira
https://issues.sonatype.org/browse/INT-536
### Screencast
https://youtu.be/-qaquP14mJw

### CODE REVIEW

File: recipes/hazelcast.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#
```
[the remark this answers is not in the document]
Comment (fernau) [reply recipes/hazelcast.rb]: this is not AWS specific (also supports static, multicast,..). so I renamed it to ha-hazelcast.
we might add HA redhat or cubernetes,...

File: test/smoke/default/configure_test.rb [hunk 2]
```diff
@@ -8,63 +8,6 @@
...
-
-  context 'when group name is set' do
-    its('content') { should include '<name>nexus</name>' }
-  end
-
-  context 'when group password is set' do
-    its('content') { should include '<password>nexus123</password>' }
-  end
-
-  context 'when multicast is enabled' do
```
Comment (bigspotteddog) [inline test/smoke/default/configure_test.rb]: Looks like we dropped tests for some of these? They are still in the template.

- multicast enabled
- tcp-ip disabled
- aws disabled
- iam_role
- region
- tag key/value
- hazelcast discovery disabled

Do we not need to test these?

Comment (fernau) [reply test/smoke/default/configure_test.rb]: these tests are just removed from this place, but they are duplicated across all the other tests. for example (not exclusively):

multicast enabled is here: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/multicast_enabled/configure_test.rb#L13

tcp-ip disabled: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/multicast_enabled/configure_test.rb#L29

aws disabled: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/multicast_enabled/configure_test.rb#L25

iam_role: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/aws_enabled/configure_test.rb#L13

region: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/aws_enabled/configure_test.rb#L21

tag: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/aws_enabled/configure_test.rb#L25

discovery disabled: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/aws_enabled/configure_test.rb#L41

Comment (bigspotteddog) [reply test/smoke/default/configure_test.rb]: Great! Thanks!

**Pull request merged**